extraTs/utils/Permissions.ts

```diff
@@ -0,0 +1,17 @@
+import { permissions } from '../constant';
+export default function hasPermission( moduleName: string , role: string , permissionType: string)
+{
+    console.log(permissions);
+    console.log(moduleName);

```

Comment:
Enter return type of function

Reply:
I forgot to write the return type of function in this files.